### PR TITLE
BaSP-MR-318: Fix subs create and edit

### PR DESCRIPTION
**Pull request by FrancoLelli**
I fixed the subscription form. Now you can add or remove one or more members.

- [X] Create array to each member joined to class.
- [X] Add some way to see and edit only one member

### CODE REVIEW

**Comment by FrancoLelli** [comment]
[Fix/update subscription](https://trello.com/c/7dfUfUIp/318-fix-update-subscription)

File: src/Components/Subscriptions/FormSubscription/Index.jsx [hunk 1]
```diff
@@ -45,7 +56,7 @@ const FormSubscription = () => {
   const subscriptionUpdated = {
     classId: data.classId?._id,
     members: data.members?.map((member) => member._id),
-    date: data.date && new Date(data.date).toISOString().substr(0, 10)
```
Comment (maurojjzz) [inline src/Components/Subscriptions/FormSubscription/Index.jsx:48]: Don't delete `&& new Date(data.date).toISOString().substr(0, 10)`.
image


 This brings the actual date to the Date Input . Without it it shows like on the photo above.

**Review by maurojjzz — CHANGES_REQUESTED** [review]
requested a change. Other than that the rest looks great

File: src/Components/Subscriptions/FormSubscription/Index.jsx (hunk 1 above)
Comment (FrancoLelli) [reply src/Components/Subscriptions/FormSubscription/Index.jsx]: Ok!

**Review by maurojjzz — APPROVED** [review]

**Review by CristianLotorto — APPROVED** [review]

**Pull request merged**